web/src/views/LoginPortal/SecondFactor/MethodContainer.tsx

```diff
@@ -27,12 +28,13 @@ export interface Props {
 
 const DefaultMethodContainer = function (props: Props) {
     const style = useStyles();
+    const { t } = useTranslation("Portal");
```

Comment:
```suggestion
    const { translate } = useTranslation("Portal");
```

Can you move this const up a level so you don't have to define it multiple times in the different areas?